.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: AI Code Reviewer
+
+on: [pull_request]
+
+    # types:
+    #   - opened
+    #   - synchronize
+# permissions: write-all
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        
+      - name: Read Review Guidelines
+        id: read-guidelines
+        run: |
+          # Assuming your guidelines are in a file named REVIEW_GUIDELINES.md
+          REVIEW_GUIDELINES=$(cat REVIEW_GUIDELINES.md | base64)
```

Comment:
It's a good practice to check if the file exists before trying to read it. Consider adding a check for the existence of the REVIEW_GUIDELINES.md file.